fhir-server-test/src/test/java/com/ibm/fhir/server/test/bulkdata/ExportOperationTest.java

```diff
@@ -349,8 +351,21 @@ public CloseableHttpClient getHttpClient() {
                 .build();
     }
 
-    public void verifyFileLines(String workItem, int count) throws IOException {
-        int actual = Files.readAllLines(new File(path + "/" + workItem).toPath()).size();
+    public void verifyFileLines(String workItem, int count, String resourceType) throws IOException {
+        List<String> lines = Files.readAllLines(new File(path + "/" + workItem).toPath());
+        for (String line : lines) {
+            try (ByteArrayInputStream bais = new ByteArrayInputStream(line.getBytes())) {
+                try {
+                    // Checks to see if it's a single type.
+                    com.ibm.fhir.model.resource.Resource r = FHIRParser.parser(Format.JSON).parse(bais);
+                    assertEquals(resourceType, r.getClass().getSimpleName());
+                } catch (FHIRParserException e) {
+                    e.printStackTrace();
+                    Assert.fail();
+                }
+            }
```

Comment:
its just test code, but I think we can have a single try here:
```suggestion
            try (ByteArrayInputStream bais = new ByteArrayInputStream(line.getBytes())) {
                // Checks to see if it's a single type.
                com.ibm.fhir.model.resource.Resource r = FHIRParser.parser(Format.JSON).parse(bais);
                assertEquals(resourceType, r.getClass().getSimpleName());
            } catch (FHIRParserException e) {
                e.printStackTrace();
                Assert.fail();
            }
```